css/global.css

```diff
@@ -1,6 +1,26 @@
 button:has(div):has(div):has(div):has(div):has(svg[aria-label="Split stay"]),
+button:has(div):has(div):has(div):has(div):has(
+    svg[aria-label="Estancia combinada"]
+  ),
+button:has(div):has(div):has(div):has(div):has(
+    svg[aria-label="Estancia conectada"]
+  ),
+button:has(div):has(div):has(div):has(div):has(
+    svg[aria-label="Verbundener Aufenthalt"]
+  ),
+button:has(div):has(div):has(div):has(div):has(
+    svg[aria-label="Soggiorni collegati"]
+  ),
+button:has(div):has(div):has(div):has(div):has(
+    svg[aria-label="Séjour en deux temps"]
+  ),
+button:has(div):has(div):has(div):has(div):has(
+    svg[aria-label="Estadias combinadas"]
+  ),
 div[data-testid="split-stays"],
 .t1wk5n1c,
-.sbylqvx {
+.sbylqvx,
+.sffcsco,
+.ht87d5e {
```

Comment:
These two class names reference the heading and the actual split stay section on the left hand side of the page.